userspace/engine/falco_engine_version.h

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // The version of rules/filter fields/etc supported by this falco
 // engine.
-#define FALCO_ENGINE_VERSION (6)
+#define FALCO_ENGINE_VERSION (7)
```

Comment:
I see that FALCO_FIELDS_CHECKSUM was already updated with the new checksum, great. Generally, whenever that checksum changes, the engine version needs to change too.